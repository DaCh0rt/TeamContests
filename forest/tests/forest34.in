539 886
-9547 9859
-9849 -8846
6821 3604
-5083 7958
1792 6448
-974 2422
-6014 6521
-1217 -5659
-4574 -7691
591 5827
3981 7187
4070 4859
9809 -6050
-2053 -3247
-2930 -9167
-7254 7465
-1407 -5280
-9221 -4962
-8412 5578
-6653 -9490
-6376 -7784
-3476 8768
6336 4189
3504 6457
2170 3938
-71 -2972
-7057 -2751
5181 -7924
-6153 6460
-5453 -6039
1604 -1570
-9069 -300
9153 -9548
7273 -6647
-6208 440
3764 8399
-2007 -1202
-2006 8365
3224 -2436
1977 8332
-4502 -683
-109 6054
1835 -8962
-6850 -5424
-3922 653
6667 -689
-6931 9969
7265 9942
-3671 -4838
4463 -7727
-48 6164
-1393 1738
4091 -9323
288 -9189
-4893 283
339 -4371
-9621 4173
-4136 -7581
7078 -9095
8409 6796
968 -2104
-8523 26
-5961 -8405
2695 6064
-3238 3957
-9362 -3727
7074 761
2369 391
-2183 7014
8076 9889
-5111 4359
-7177 727
7079 -2581
4425 -3917
-6736 -6946
-3476 9704
8160 -9322
-9238 -3903
4646 4889
-5695 9566
5904 -7342
9710 4097
-986 -9145
5881 -2196
3857 8316
-1688 -6354
-1743 5087
-6501 112
-4922 -9688
3217 -9015
1411 5078
-5758 -3269
-835 -1740
4762 -2584
-5508 -1168
-8462 4508
4252 -7767
8522 -163
-3738 -7987
3181 3858
741 -3418
5353 -4444
-8286 4241
-6790 -8539
951 8239
4087 445
7018 5733
-9158 -3953
-566 7573
-4841 5300
4743 -3179
8249 -4739
4335 2344
-3068 -472
1140 6396
7710 3475
-1225 -207
4847 1059
-125 -2750
7700 -5154
8605 -4891
8027 7056
1066 -7757
2512 8018
4716 -3396
-6021 -3238
-1329 738
-3203 1107
3698 -1964
1457 5016
2103 949
4857 8920
8701 -6848
9526 1691
-2332 2349
7593 -730
-2608 -1549
-3902 5893
944 5412
-6950 2543
-4208 9351
48 -2073
2546 -6268
-2983 -6636
2160 -2478
-9380 -4070
5204 -4618
-8617 4023
-6220 3586
1157 -9876
8151 9150
-473 7801
-6917 -4269
9995 -3045
8907 2983
-8923 2322
8920 -4642
-4440 1528
7026 3076
-4408 1623
5921 -4064
-1533 -8611
4586 -8069
-4037 -5416
-8914 -660
-6096 -6413
-1326 5797
3682 -9611
-9505 -5815
-305 20
1744 -6755
4112 -2498
9875 6206
5013 -2063
-792 -8787
3457 -8796
-9340 9981
3517 2245
-9385 -4782
-8946 -4780
8222 8737
5144 -2268
394 -7884
-5026 -8954
-6910 8392
6982 9085
5996 -4707
-964 -9700
8413 -1145
-4134 -1475
6858 -5705
12 -8638
1533 8207
7214 -6833
-7122 -8495
-7059 -4897
-6440 -3930
9601 -5740
1376 -4785
9966 1216
9545 5435
-6727 5991
1412 -4608
-3415 6694
4553 8821
-1137 -4897
-5020 -8029
-3411 -1098
-7824 3658
-2812 -21
8624 -7270
-1165 -6640
866 -1251
-8974 -4249
1205 -9419
-7624 -9355
6088 9273
4457 1749
-6987 8212
430 3528
8317 -3222
6346 3462
6986 -9929
4992 5078
-1984 -7827
633 8235
-2190 -6426
1821 9883
8508 -2721
9847 9756
-3319 -2718
6115 9437
7655 -7798
-5269 8199
-617 2233
4293 -4407
3550 -6853
6033 -9047
-2799 -3723
6062 9378
-244 7014
6385 3784
8299 2773
7358 -289
-8686 -6727
1756 -9924
-6038 -3342
1846 6260
2261 -1893
4058 8411
4507 -8283
7720 -3478
7152 3268
1973 -6521
-6888 -4270
-2519 4297
-8564 6382
-5316 3977
-2996 -1093
46 -1978
6425 8545
7475 -9489
-1563 -8831
-9991 -3257
7588 -4711
7265 9062
-4879 -568
819 -365
6626 -3383
-7636 -2170
7284 9076
3360 4243
2090 -5446
4475 7262
-935 6486
-8300 -356
-7713 4414
3133 792
-5630 -9805
-6540 1275
4603 302
-9493 -6355
-1451 -4931
3755 -6666
-8386 -9294
7420 -9414
8792 -499
-9150 -4895
-2907 4662
-2132 -969
-4276 -1110
-6101 -9433
-6319 555
-741 -5845
6002 -8960
8398 4552
5023 4425
-7446 1469
-5956 3997
4584 -4406
2349 8307
1711 7045
4904 -7975
1455 -3575
7325 -8234
-4798 2809
-5922 -5579
-3092 722
4120 9814
3334 -5030
-3779 3605
2312 -4831
6556 -8595
4685 -4446
-3587 -6340
-306 -3647
-2951 -1629
2728 -138
3056 -2945
-1910 3849
-4605 -2727
-6897 5037
1328 3799
-5096 8008
-253 1932
2228 9463
-1417 2014
1574 -4667
-6076 3449
-2910 6617
-7323 5998
7072 7850
-8976 6480
771 3423
-3498 -5776
-5353 6081
-483 2316
-263 -6689
-4277 -2390
-7834 8607
-2939 -2463
4763 2439
3924 -7452
8608 -173
-4273 9279
768 -9837
-9680 6229
-4834 -7052
8668 4933
-5945 4938
1498 -7991
-8061 8236
2157 -6220
-8168 -7057
5755 908
1018 -6679
9897 862
7037 -7701
9779 -884
-9199 9391
-8889 8337
6492 -3517
-6051 -4101
-2048 9392
-1126 -9985
165 8202
3726 -1619
2781 -7825
-5090 9249
6103 -36
-7506 7828
7485 3410
-4991 -8271
-827 -9753
8372 -5715
2760 6305
1351 4601
-2811 3563
3104 -4152
7723 7536
9795 4681
5873 1706
-2390 -1979
-8546 -7532
-9432 3997
6478 -8426
-9923 -5770
-4582 333
189 -1743
-9252 -8162
-9560 8756
361 6765
-8221 -4265
4502 5598
6180 5888
5513 4582
4591 -8277
7966 6691
-2343 4245
-2807 -9335
-2229 1739
-2849 517
-1010 1455
8221 321
-1580 -4300
7350 -335
-9636 6710
-4341 3429
3464 159
-1103 -9657
-9957 -2365
-1377 -1247
-1623 9788
7091 -8214
4233 4896
1639 -9496
-8771 4907
-9869 -5922
3715 453
-4431 -2933
-7007 7981
-3961 1413
-4 4623
4400 3023
4905 811
1439 -3918
445 2081
4667 -8052
3193 -8445
-313 -3266
4307 8032
-1172 5878
-3769 -5425
2736 2483
6891 -6766
7879 -7631
-120 9154
8929 -7118
-5246 -751
-6558 6508
3116 6273
2326 -139
5918 6670
9712 6961
839 1129
7428 -1773
2874 7849
-1652 6644
3347 1496
8310 8092
873 5840
4560 1673
5462 -5974
-4248 -7413
-6887 808
4902 699
-7239 -1921
-2328 6353
8841 -5506
3600 1894
-7622 -5
2475 -8372
3053 -6283
-6074 -2244
4986 -1809
-1909 1380
-3218 -3762
1864 1970
-6771 7448
3387 -7987
8537 3666
6445 3671
7694 -3804
-7970 -3125
2065 -5446
6272 -8599
8698 -4186
-5967 -3620
-9253 8207
-3547 -8243
-6924 -8114
-9279 -4078
-7600 -3408
-7588 2913
7177 320
1891 -4986
5345 319
5267 -3471
4734 -3971
-4268 -7176
2311 -6313
5318 27
-2144 -2965
7743 -4104
-5627 8074
5126 -3896
312 -3404
2211 6794
-1420 7942
-2244 -1284
-8826 -3978
-7746 1795
-4855 -840
1508 6090
-9850 -7599
-1238 4256
2311 3899
-9548 9197
-5927 4314
9712 1955
-3057 -447
-6089 5931
-8073 -4300
-6098 -9527
-3714 2780
947 -6157
-3683 -4789
8391 9938
-7024 -4272
-1505 6924
3939 -2085
4526 -6277
-4091 -1353
-6864 4672
5869 581
-7425 -9651
-1074 -3837
4220 7317
-6150 -1077
-8227 -3000
6989 -5256
-1516 4074
9692 -7133
4676 7431
9511 3126
1382 -8367
9961 2131
-2544 -4996
2447 5471
