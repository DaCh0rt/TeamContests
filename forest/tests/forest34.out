82272.13 458908406.86
